src/main/resources/org/biouno/unochoice/stapler/unochoice/UnoChoice.es6

```diff
@@ -46,6 +46,13 @@ var UnoChoice = UnoChoice || ($ => {
     let instance = {};
     let SEPARATOR = '__LESEP__';
     let cascadeParameters = [];
+    let loadingCounter = 0;
+    function incrementLoadingCounter() {
+        loadingCounter++;
+    }
+    function decrementLoadingCounter() {
+        loadingCounter--;
+    }
```

Comment:
I thought it was going to be used to check for consistency, or maybe for the tests, but I couldn't locate where the `loadingCounter` value is being read. Is that somewhere in this file? Maybe GitHub UI is not showing it in the diff (didn't have time to check it out tonight, sorry).

Reply:
I did try it. I got rid of it. I'll remove this code. Thanks for the extra pair of eyes on this.